mtp_noms_ops/apps/settings/forms.py

```diff
@@ -264,3 +264,47 @@ def get_confirmation_query_string(self):
         query_dict = self.request.GET.copy()
         query_dict['prisons'] = prisons
         return urlencode(query_dict, doseq=True)
+
+
+class JobInformationForm(ApiForm):
+    job_titles = [
+        ('Evidence collator', _('Evidence collator')),
+        ('Intelligence analyst', _('Intelligence analyst')),
+        ('Intelligence officer', _('Intelligence officer')),
+        ('Intelligence researcher', _('Intelligence researcher')),
+        ('Intelligence support officer', _('Intelligence support officer')),
+        ('Safety analyst', _('Safety analyst')),
+        ('Security analyst', _('Security analyst')),
+        ('Other', _('Other'))
+    ]
+
+    prison_estates = [
+        ('Local prison', _('Local prison')),
+        ('Regional', _('Regional')),
+        ('National', _('National')),
+    ]
+    job_title = forms.ChoiceField(label=_('What is your job title?'),
+                                  choices=job_titles,
+                                  widget=forms.RadioSelect)
+    prison_estate = forms.ChoiceField(label=_('Which area of the prison estate do you work in?'),
+                                      choices=prison_estates,
+                                      widget=forms.RadioSelect)
+    tasks = forms.CharField(label=_('What are your main tasks?'),
+                            widget=forms.Textarea,
```

Comment:
as it happens, the `widget` argument won't have any effect because of how our html templates work. no harm having it tho

Reply:
If it's not used, I'd rather remove it,  to avoid confusion and an assumption it is used, good spot.